app/src/types/articles.ts

```diff
@@ -0,0 +1,22 @@
+type Source = {
+  id: string | null;
+  name: string;
+};
+
+export type Article = {
+  source: any;
+  author: string | null;
+  title: string;
+  description: string;
+  url: any;
```

Comment:
As per the API response, seems that url and urlToImage attributes are always a string, we could replace any by string